guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -38,7 +38,7 @@ endif::[]
 * On a virtual machine, take a snapshot.
 * On a physical machine, create a backup.
 +
-For information on backups, see {AdministeringDocURL}Backing_Up_Server_and_Proxy_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in the _Administering {ProjectName} {ProjectVersionPrevious}_ guide.
+For information on backups, see {AdministeringDocURL}Backing_Up_Server_and_Proxy_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
```

Comment:
The original title is weird to me.

Reply:
It definitely is weird, but the replacement is correct :D